_posts/en/newsletters/2021-09-22-newsletter.md

```diff
@@ -0,0 +1,165 @@
+---
+title: 'Bitcoin Optech Newsletter #167'
+permalink: /en/newsletters/2021/09/22/
+name: 2021-09-22-newsletter
+slug: 2021-09-22-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes a proposed modification to the BIPs
+process, summarizes a plan to add support for package relay to Bitcoin
+Core, and links to a discussion about adding LN node information to DNS.
+Also included are our regular sections with descriptions of changes to
+services and client software, how you can prepare for taproot, new
+releases and release candidates, and notable changes to popular Bitcoin
+infrastructure software.
+
+## News
+
+- **BIP extensions:** Karl-Johan Alm [posted][alm bips] to the
+  Bitcoin-Dev mailing list a proposal that BIP documents which have
```

Comment:
I tripped over "a proposal that BIP documents which ..." a bit, I think because my brain interpreted _documents_ as a verb. I think perhaps "a proposal that BIPs which ..." might be clearer.